src/ProductTour/CheckpointHeader.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import Icon from '../Icon';
+import IconButton from '../IconButton';
+import { Close } from '../../icons';
+import CheckpointTitle from './CheckpointTitle';
+import messages from './messages';
+
+const CheckpointHeader = React.forwardRef(({
+  index, onDismiss, title, totalCheckpoints,
+}) => {
+  const intl = useIntl();
+  const oneBasedIndex = index + 1;
+
+  const closeAltText = intl.formatMessage({
+    id: 'pgn.ProductTour.checkpointHeader.close',
+    defaultMessage: 'Close tour',
+    description: 'Close alt text for ProductTour component',
+  });
+
+  return (
+    <>
+      <div className="pgn__checkpoint-header">
+        <p className="pgn__checkpoint-page-index">
+          <FormattedMessage
+            id="pgn.ProductTour.CheckpointHeader.pageIndex"
+            {...messages.pageIndexText}
+            values={{ step: oneBasedIndex, totalSteps: totalCheckpoints }}
+          />
+        </p>
+        <IconButton
+          iconAs={Icon}
+          src={Close}
+          alt={closeAltText}
+          onClick={onDismiss}
+          variant="primary"
```

Comment:
Do we need to specify the "primary" variant if it is specified by default for buttons in Paragon?